src/Translator.php

```diff
@@ -85,13 +91,6 @@ private function checkEnv(): void
         } catch (ProcessFailedException $e) {
             throw new RuntimeException('Could not find wget command. Please install it.');
         }
-        $this->txSite = getenv('TX_SITE');
-        if (!$this->txSite) {
-            throw new RuntimeException('TX_SITE environment variable is not defined');
-        }
-        if (strpos($this->txSite, 'http://') !== 0 && strpos($this->txSite, 'https://') !== 0) {
-            $this->txSite = 'http://' . $this->txSite;
-        }
         if (!parse_url($this->txSite, PHP_URL_HOST)) {
             throw new RuntimeException('SITE environment variable is not a valid url');
         }
```

Comment:
Sorry, I thought I had captured this part in my previous review as well. This also needs to be moved to after the assignment - and probably only be done if `$this->doCollectStrings` is truthy.

Reply:
Updated